snapcraft/plugins/catkin.py

```diff
@@ -79,14 +79,17 @@
 import shutil
 import subprocess
 import textwrap
-from typing import List, Set
+from typing import List, Set, TYPE_CHECKING
 
 import snapcraft
 from snapcraft.plugins import _ros
 from snapcraft.plugins import _python
 from snapcraft import common, file_utils, formatting_utils, repo
 from snapcraft.internal import errors, mangling
 
+if TYPE_CHECKING:
+    from snapcraft.project import Project
```

Comment:
This is odd. It wasn't required before we added rospack... not sure why it's needed now. :man_shrugging: .

Reply:
rospack had this import in it before

Reply:
But we didn't need it BEFORE we added rospack either. Anyway, no changed needed here, I just find it confusing and wanted to whine.

Reply:
Been needed since: https://github.com/snapcore/snapcraft/commit/d1c5f1b00abe312b7df6e5a7c84896de5657c992#diff-3cc26b49a70ee86523598a96dd385301R1081

Reply:
Nice detective work. That makes more sense.